packages/store/src/Hook.sol

```diff
@@ -30,15 +30,15 @@ library HookLib {
    * @notice Filter a hook from the hook list by its address.
    * @dev This function writes the updated hook list to the table in place.
    * @param hookTableId The resource ID of the hook table.
-   * @param tableWithHooks The resource ID of the table with hooks to filter.
+   * @param elementWithHooks The resource ID of the table with hooks to filter.
```

Comment:
can we name this `resourceWithHooks`?

"element" is a term we've never used before and I think "resource" is more consistent: https://www.notion.so/latticexyz/MUD-naming-conventions-terminology-66e11356a1934465b2aa1a951c61a299